index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gozo Browser Extension</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="app">
+
+        <div class="weather">
+            <div class="icon"><img src="sun.svg" alt="sun icon"></div>
+            <div class="temp">32<span>&#176;</span></div>
+            <div class="loca">Ahmedabad</div>
+        </div>
+
+        <div class="center">
+
+            <div class="clock">
+                <h1>10:29</h1>
+            </div>
+
+            <div class="greeting">
+                <h1>Good Morning, Maitra !!</h1>
+            </div>
+
+            <div class="main-focus">
```

Comment:
please remove unused code and do proper indentation

Reply:
I'll take care of indentation, and actually that is not unused code, it will be based on conditional rendering so I comented it